sopel/plugin.py

```diff
@@ -1248,10 +1298,30 @@ def rate_global(
     :param message: optional; message sent as NOTICE when a user hits the limit
 
     This decorator can be used alone or with the :func:`rate` decorator, as it
-    will always take precedence::
+    will always take precedence.
+
+    The message can contain placeholders which will be filled in:
+
+        - ``nick``: the nick that hit the rate limit
+        - ``channel``: the channel in which the rate limit was hit (will be
+          ``'private-message'`` for private messages)
+        - ``sender``: the sender (nick or channel) of the message which hit
+          the rate limit
+        - ``plugin``: the name of the plugin that hit the rate limit
+        - ``label``: the label of the plugin handler that hit the rate limit
+        - ``time_left``: the time remaining before the rate limit expires, as
+          a string
+        - ``time_left_sec``: the time remaining before the rate limit expires,
+          expressed in number of seconds
+        - ``rate_limit``: the rate limit, as a string
+        - ``rate_limit_sec``: the rate limit, expressed in number of seconds
+        - ``rate_limit_type``: the type of rate limit that was hit (one of
+          ``user, group, global``)
```

Comment:
4th times the charm!

That makes me wonder: if they are all the same, maybe reference them only one, and use a sentence like:

```
The message can contain placeholders which will be filled in; they are the same
placeholders as :func:`that_other_rate_function`.
```

Reply:
I was worrying about the repetition but this straightforward approach did not occur to me. Thanks, done in `ae9a96d`